src/features/commands.ts

```diff
@@ -567,6 +567,53 @@ Here's an article explaining the difference between the two: https://goshakkk.na
       });
     },
   },
+  {
+    words: [`!jwt`],
+    help: `Describes reasoning for and against the use of JWT tokens againt using standard sessions`,
+    category: "Reactiflux",
+    handleMessage: (msg) => {
+      msg.channel.send({
+        embeds: [
+          {
+            title:
+              "Is JWT the right approach for my applications authentication?",
+            type: "rich",
+            description: `
+            **TL;DR: JWT is widely overused as the go to authentication choice over sessions. Use this bot to form your opinon on wether it is a good fit for you.**
+            
+            A lot of developers starting their journey into programming will, at some point, hit the question of how to work with backend authentication.
+            They will undoubtably come across the multitude of JWT tutorials on the subject. But is this the correct approach?. Most of the time the answer will be no. 
```

Comment:
Redundant dot at "approach?**.**".
I would also wrap "no" in quotes in "will be **no**".